src/public/publicAPIs.ts

```diff
@@ -312,4 +312,31 @@ export function navigateToTab(tabInstance: TabInstance, onComplete?: (status: bo
 
   const errorMessage = "Invalid internalTabInstanceId and/or channelId were/was provided";
   GlobalVars.callbacks[messageId] = onComplete ? onComplete : getGenericOnCompleteHandler(errorMessage);
-}
\ No newline at end of file
+}
+
+/**
+ * @private
+ * Hide from docs until release.
+ * ------
+ * Sends query to bot in order to retrieve data.
+ * @param botRequest query to send to bot.
+ * @param onBotResponse callback to invoke when data is retrieved from bot
+ * @param onError callback to invoke should an error occur
+ */
+export function sendBotRequest(botRequest: BotRequest, onBotResponse?: (data: BotResponse) => void, onError?: (error: string) => {}): void { // void for now
+    ensureInitialized();
+
+    // send request to teams
+    const messageId = sendMessageRequest(GlobalVars.parentWindow, "executeBotQuery", [
+      botRequest
+    ]);
+
+    // register handler for callback id
```

Comment:
Lets put bot related functions and interfaces inside its own namespace (take a look on task.ts file)

Reply:
export namespace bot {
    sendQueryRequest(botRequest: BotRequest, onBotResponse?: (data: string | BotResponse) => void, onError?: (error: string) => {}): void {
  ....
// send request to teams
    const messageId = sendMessageRequest(GlobalVars.parentWindow, "bot.executeQuery", [
      botRequest
    ]);

...
}